src/components/molecules/DropdownFiltered/DropdownFiltered.tsx

```diff
@@ -44,10 +44,9 @@ export const SearchInputWrap = styled.div`
 
 export const CustomIcon = styled.div<{ isOpen: boolean }>`
   background: transparent url(${unbrandedChevron}) no-repeat center;
-  background-size: 60%;
+  background-size: 120%;
```

Comment:
don't think this change has affected anything anywhere else but cant be sure - struggled passing theming into this component for some reason